specification/assets/gherkin/hooks.feature

```diff
@@ -0,0 +1,52 @@
+@hooks
+Feature: Evaluation details through hooks
+
+# This test suite contains scenarios to test the functionality of hooks.
+
+  Background:
+    Given a provider is registered with cache disabled
+
+  Scenario: Passes evaluation details to after and finally hooks
+    Given a client with added hook
+    And a boolean-flag with key "boolean-flag" and a default value "false"
+    When the flag was evaluated with details
+    Then "before" hooks should be called
```

Comment:
is it one hook? or multiple ones, for me it looks more like 
```
Then the "before" hook should be called
```

of to be more in sync with the already existing flagd feature files (https://github.com/open-feature/flagd-testbed/blob/main/gherkin/events.feature#L27)

```
Then the "before" hook should have been executed
```

but i am not sure about this naming